server/src/routes/api/v1/showRouter.js

```diff
@@ -0,0 +1,25 @@
+import express from "express"
+// import { Show } from 
```

Comment:
no code comments, you need to actually create the Show model and migration for your database for this PR to be accepted